core-parser/src/main/java/org/uniprot/core/parser/tsv/uniprot/EntryReferenceMap.java

```diff
@@ -26,16 +26,20 @@ public Map<String, String> attributeValues() {
             return Collections.emptyMap();
         }
 
-        String result =
+        Map<CitationDatabase, String> idMaps =
                 references.stream()
                         .map(UniProtKBReference::getCitation)
                         .filter(val -> val.getCitationCrossReferences() != null)
                         .flatMap(val -> val.getCitationCrossReferences().stream())
-                        .filter(val -> val.getDatabase().equals(CitationDatabase.PUBMED))
-                        .map(CrossReference::getId)
-                        .collect(Collectors.joining("; "));
+                        .collect(
+                                Collectors.groupingBy(
+                                        CrossReference::getDatabase,
+                                        Collectors.mapping(
+                                                CrossReference::getId, Collectors.joining("; "))));
+
         Map<String, String> map = new HashMap<>();
-        map.put(FIELDS.get(0), result);
+        map.put(FIELDS.get(0), idMaps.getOrDefault(CitationDatabase.PUBMED, ""));
```

Comment:
I would prefer these strings just as 2 constants, rather than having them inside a list. It's easier to refer.
What is the advantage of having them inside a list?